pkg/media/whip/appsrc.go

```diff
@@ -174,7 +170,6 @@ func (w *whipAppSource) copyRelayedData(r io.Reader) error {
 		}
 
 		b := gst.NewBufferFromBytes(re.data)
-		b.SetPresentationTimestamp(gst.ClockTime(re.ts))
```

Comment:
Without a timestamp, how does the gstreamer pipeline generate PTS? I believe this PR entirely removes the synchronizer. We added it (and iterated extensively on it) initially in the egress server because we needed to generate a somehow trustworthy PTS timeline to send data to gstreamer, and keep proper pacing and A/V sync. Experience shows that we probably need the synchronizer (and jitter buffer) in the gstreamer path. It should probably be moved to the handler process/media path to keep it out of the bypass_transcoding workflow.

Reply:
Specifically, since we do not forward RTCP from the WHIP sender to the SFU, and because the audio and video pipeline, paths are likely to have a different latency, I believe we need to reconstruct a full media timeline in ingress to be able to keep proper A/V sync